scripts/build_docker_image.sh

```diff
@@ -2,7 +2,21 @@
 
 set -euo pipefail
 
-# Avalanche root directory
+# If set to non-empty, prompts the building of a multi-arch image when the image
```

Comment:
not sure if this file is used at all?

Reply:
No, I don't think it's used in Coreth as Coreth packed with AvalancheGo's docker.

Reply:
Let's remove it.